.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -0,0 +1,314 @@
+      id: set-manual-url
+      run: |
+        if [ "${{ github.event_name }}" == "schedule" ]; then
+          # Scheduled workflows should use the master branch
+          echo "manual_url=https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        else
+          # Determine the SHA to use
+          if [ -z "${{ github.event.pull_request.head.sha }}" ]; then
+            # Fallback to the current ref or SHA if it's not a pull request event
+            sha=${{ github.sha }}
+          else
+            sha=${{ github.event.pull_request.head.sha }}
+          fi
+
+          echo "manual_url=https://raw.githubusercontent.com/stackrox/stackrox/$sha/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        fi
+
+    - uses: ./.github/actions/upload-artifact-with-retry
+      with:
+        name: nvd
+        path: nvd.zip
+        if-no-files-found: error
+
+  build-and-run:
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    needs:
+    - parse-versions
+    - prepare-environment
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.2
+      volumes:
+      # The updater makes heavy use of /tmp files.
+      - /tmp:/tmp
+      - /usr:/mnt/usr
+      - /opt:/mnt/opt
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        include: ${{ fromJson(needs.parse-versions.outputs.versions) }}
+    env:
+      SCANNER_BUNDLE_VERSION: ${{ matrix.version }}
+      ROX_GIT_REF: ${{ matrix.ref }}
+    steps:
+    - name: Free up disk space
+      shell: bash
+      run: |
+        set +e
+        set -x
+        df -h
+        for delete in /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache/CodeQL; do
+          rm -rf "/mnt${delete:?}"
+        done
+        df -h
+
+    - name: Checkout specific reference
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: ${{ env.ROX_GIT_REF }}
+
+    - uses: ./.github/actions/job-preamble
+      with:
+        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+
+    - name: Build updater
+      run: |
+        echo "Building updater for version ${{ env.SCANNER_BUNDLE_VERSION }} on branch ${{ env.ROX_GIT_REF }}..."
+        make tag
+        make -C scanner bin/updater
+
+    - name: Download NVD
+      uses: ./.github/actions/download-artifact-with-retry
+      with:
+        name: nvd
+        path: nvd.zip
+
+    - name: Set Updater
+      run: |
+        chmod +x scanner/bin/updater
+        mkdir vulns
+
+    - name: Create Folder
+      run: mkdir -p definitions/${{ env.SCANNER_BUNDLE_VERSION }}
+
+    - name: Run Updater (single bundle)
+      env:
+        STACKROX_NVD_ZIP_PATH: nvd.zip
+      run: |
+        scanner/bin/updater export vulns --manual-url "${{ needs.prepare-environment.outputs.manual_url }}"
+        mv vulns/* definitions/${{ env.SCANNER_BUNDLE_VERSION }}/
```

Comment:
@daynewlee Sorry Yi, short notice on this one; But we don't need to support multi-bundles 4.5 or dev. I am not sure how much support we still want to offer for 4.4, but creating single-bundle for 4.4 for now is fair.

CC @RTann

Reply:
not for dev? But dev reflects changes in master, it should be the latest, no?
I think for v1 we can generate both and when we update release-vuln workflow, we don't need to copy both.

Reply:
I will make the following changes in next PR for release vuln bundle workflow:
4.4 single
4.5 single and multi
4.6 & dev multi